ui/ops/src/routes/ops/security-events/SecurityEventsTable.vue

```diff
@@ -1,24 +1,23 @@
 <template>
   <div class="ml-2">
-    <v-row v-if="!props.overviewPage" class="mt-0 ml-0 pl-0">
+    <v-row class="mt-0 ml-0 pl-0">
       <h3 class="text-h3 pt-2 pb-6">Security Events</h3>
       <v-spacer/>
     </v-row>
 
-    <content-card :class="['px-8', {'mt-8 px-4': !props.overviewPage}]">
+    <content-card :class="['px-8', 'mt-8 px-4']">
```

Comment:
If `class` isn't dynamic, probably best to just write it `class="px-8 mt-8 px-4"` which immediately shows that `px-8` and `px-4` are conflicting styles and should be fixed.